### PR TITLE
Updated Configuring section to instruct users to modify json files, not Python

### PR DESCRIPTION
**Pull request by mhoeber**
### Reviewers
- [ ] Subject matter expert:  @e0d 
- [ ] Subject matter expert: 
- [ ] Doc team review (copy) @lamagnifica : 
- [ ] Product review:
- [ ] Partner support: 
- [ ] PM review: 
### Testing
- [x] Ran en_us/make without warnings or errors
### Post-review
- [x] Add description to release notes task as a comment
- [x] Update change log
- [x] Squash commits

### CODE REVIEW

File: en_us/install_operations/source/configuration/enable_certificates.rst [hunk 1]
```diff
@@ -32,19 +32,25 @@ configuration tasks described in this topic.
 Enable Certificates in Studio and the Learning Management System
 *****************************************************************
 
-#. In the ``/cms/envs/common.py`` and ``/lms/envs/common.py`` files, set the
-   value of ``CERTIFICATES_HTML_VIEW``  to ``True``.
+To enable certificates, you modify the 
+To enable prerequisite courses, you modify the ``lms.env.json`` and
```
Comment (lamagnifica) [inline en_us/install_operations/source/configuration/enable_certificates.rst]: consolidate lines 35-36

File: en_us/install_operations/source/configuration/enable_certificates.rst [hunk 2]
```diff
@@ -32,19 +32,25 @@ configuration tasks described in this topic.
...
+``cms.env.json`` files, which are located one level above the ``edx-platform``
+directory. files, which are located one level above the ``edx-platform``
+directory.
```
Comment (lamagnifica) [inline en_us/install_operations/source/configuration/enable_certificates.rst:37]: delete most of line 38 and 39

File: en_us/install_operations/source/configuration/enable_certificates.rst [hunk 3]
```diff
@@ -32,19 +32,25 @@ configuration tasks described in this topic.
...
 
-#. Save the ``/cms/envs/common.py`` and ``/lms/envs/common.py`` files.
+#. Save the ``lms.env.json`` and ``cms.env.json`` files.
 
-#. If it does not exist already, create the folder ``/tmp/certificates`` owned by
-   the user and group ``www-data``. Depending on your configuration, this folder 
-   might not survive reboots, and so might need to be created by script.
+#. If it does not exist already, create the folder ``/tmp/certificates`` owned
+   by the user and group ``www-data``. Depending on your configuration, this
+   folder might not survive reboots, and so might need to be created by script.
```
Comment (lamagnifica) [inline en_us/install_operations/source/configuration/enable_certificates.rst]: by ^a script (?)

File: en_us/install_operations/source/configuration/enable_licensing.rst [hunk 4]
```diff
@@ -36,10 +36,12 @@ in both Studio and the Learning Management System.
 Enable Licensing in Studio
 *****************************
 
-#. In the edX Platform installation directory, edit the file
-   ``/cms/envs/common.py``
+To enable licnesing, you modify the ``lms.env.json`` and
```
Comment (lamagnifica) [inline en_us/install_operations/source/configuration/enable_licensing.rst]: sp.: licensing

**Comment by lamagnifica** [comment]
a few copy-paste errors. :+1:

**Comment by mhoeber** [comment]
@lamagnifica  edits made (thanks) and Google xblock removed

**Pull request merged**